service/templates/display_title.html

```diff
@@ -5,6 +5,21 @@
 <h1 class="heading-xlarge collapse-bottom">Summary of title {{ title.number }}</h1>
 <p class="expand-bottom">This title was last changed on <b>{{ title.last_changed | date }}</b> at <b>{{ title.last_changed | time }}</b></p>
 
+<div class="download-documents text">
+  <div class="document-item">
+    <a href="{{ url_for('display_title_pdf', title_number=title.number) }}" class="image">
+      <img src="/static/images/icon-register-document.png">
```

Comment:
Yes. What would be nice is a preview image generated from the actual document. But I feel this might be unrealistic?